test/unit/org/apache/cassandra/service/DiskFailurePolicyTest.java

```diff
@@ -79,22 +88,24 @@ public DiskFailurePolicyTest(DiskFailurePolicy testPolicy, boolean isStartUpInPr
-                             { DiskFailurePolicy.stop, false, new FSReadError(new IOException(), "blah"), false, false, false},
-                             { DiskFailurePolicy.stop_paranoid, false, new FSReadError(new IOException(), "blah"), false, false, false},
-                             { Config.DiskFailurePolicy.die, false, new CorruptSSTableException(new IOException(), "blah"), false, true, false},
-                             { DiskFailurePolicy.ignore, false, new CorruptSSTableException(new IOException(), "blah"), true, false, false},
-                             { DiskFailurePolicy.stop, false, new CorruptSSTableException(new IOException(), "blah"), true, false, false},
-                             { DiskFailurePolicy.stop_paranoid, false, new CorruptSSTableException(new IOException(), "blah"), false, false, false}
+                             { die, true, new FSReadError(new IOException(), "blah"), false, true, true},
+                             { ignore, true, new FSReadError(new IOException(), "blah"), true, false, false},
+                             { stop, true, new FSReadError(new IOException(), "blah"), false, true, true},
+                             { stop_paranoid, true, new FSReadError(new IOException(), "blah"), false, true, true},
+                             { die, true, new CorruptSSTableException(new IOException(), "blah"), false, true, true},
+                             { ignore, true, new CorruptSSTableException(new IOException(), "blah"), true, false, false},
+                             { stop, true, new CorruptSSTableException(new IOException(), "blah"), false, true, true},
+                             { stop_paranoid, true, new CorruptSSTableException(new IOException(), "blah"), false, true, true},
+                             { die, false, new FSReadError(new IOException(), "blah"), false, true, false},
+                             { ignore, false, new FSReadError(new IOException(), "blah"), true, false, false},
+                             { stop, false, new FSReadError(new IOException(), "blah"), false, false, false},
+                             { stop_paranoid, false, new FSReadError(new IOException(), "blah"), false, false, false},
+                             { die, false, new CorruptSSTableException(new IOException(), "blah"), false, true, false},
+                             { ignore, false, new CorruptSSTableException(new IOException(), "blah"), true, false, false},
+                             { stop, false, new CorruptSSTableException(new IOException(), "blah"), true, false, false},
+                             { stop_paranoid, false, new CorruptSSTableException(new IOException(), "blah"), false, false, false},
+                             { best_effort, false, new FSReadError(new IOException(new OutOfMemoryError("Java heap space test")), "best_effort_oom"), true, false, false},
```

Comment:
as "Java heap space test" removed in FORCE_HEAP_OOM_IGNORE_SET , so I think some other like "Java heap space" can be used here. I don't know if this is the right suggestion.

Reply:
see above